### DISCUSSION TITLE
Send Total Power and Yield too MQTT

### DISCUSSION THREAD

**Discussion by dhaberhausen** [tbnobody/OpenDTU#858] [Ideas]
I have four micro converters and I need the total power and energy from all converters as a MQTT value.
Therefore changed the WebApi_ws_live.cpp file and set the values after calculation int the MqttHandleDtuClass.

177    MqttHandleDtu.setTotalPvYield(totalPower, totalYieldDay, totalYieldTotal);

In the MqttHandleDtuClass is publish the values with the other DTU values

        MqttSettings.publish("dtu/totalPower", String(totalPower));
        MqttSettings.publish("dtu/totalYieldDay", String(totalYieldDay));
        MqttSettings.publish("dtu/totalYieldTotal", String(totalYieldTotal));

What do you think? Is this a general good feature? Should I check it in? Or is there a better way to do it?

**Comment by tbnobody** [comment]
Thats already implemented.
https://github.com/tbnobody/OpenDTU/blob/90a0cb60c622c66f84d91bc102a5a7b424b26059/docs/MQTT_Topics.md?plain=1#L17-L27

**Reply by dhaberhausen** [reply]
OK got it. Thanks!
